app/assets/stylesheets/pages/_playbook.scss

```diff
@@ -0,0 +1,80 @@
+.pb-bg-navy { background-color: #163963;}
+.pb-footer { background-color: #cedbec; }
```

Comment:
this looks quite similar to our body background light blue -- can we standardize and use the bg-light-blue?